src/main.ts

```diff
@@ -31,23 +31,27 @@ let mainWindow: BrowserWindow | null = null
 // Check the command line arguments for a project path
 const args = parseCLIArgs()
 
-// If it's not set, scream.
-const NODE_ENV = process.env.NODE_ENV || 'production'
-if (!process.env.NODE_ENV)
-  console.warn(
-    '*FOX SCREAM* process.env.NODE_ENV is not explicitly set!, defaulting to production'
-  )
-// Default prod values
+// @ts-ignore: TS1343
+const viteEnv = import.meta.env
+const NODE_ENV = process.env.NODE_ENV || viteEnv.MODE
 
 // dotenv override when present
 dotenv.config({ path: [`.env.${NODE_ENV}.local`, `.env.${NODE_ENV}`] })
 
-process.env.VITE_KC_API_WS_MODELING_URL ??=
-  'wss://api.zoo.dev/ws/modeling/commands'
-process.env.VITE_KC_API_BASE_URL ??= 'https://api.zoo.dev'
-process.env.VITE_KC_SITE_BASE_URL ??= 'https://zoo.dev'
-process.env.VITE_KC_SKIP_AUTH ??= 'false'
-process.env.VITE_KC_CONNECTION_TIMEOUT_MS ??= '15000'
+// default vite values based on mode
+process.env.NODE_ENV ??= viteEnv.MODE
+process.env.DEV ??= viteEnv.DEV + ''
+process.env.BASE_URL ??= viteEnv.VITE_KC_API_BASE_URL
+process.env.VITE_KC_API_WS_MODELING_URL ??= viteEnv.VITE_KC_API_WS_MODELING_URL
+process.env.VITE_KC_API_BASE_URL ??= viteEnv.VITE_KC_API_BASE_URL
+process.env.VITE_KC_SITE_BASE_URL ??= viteEnv.VITE_KC_SITE_BASE_URL
+process.env.VITE_KC_SITE_APP_URL ??= viteEnv.VITE_KC_SITE_APP_URL
+process.env.VITE_KC_SKIP_AUTH ??= viteEnv.VITE_KC_SKIP_AUTH
+process.env.VITE_KC_CONNECTION_TIMEOUT_MS ??=
+  viteEnv.VITE_KC_CONNECTION_TIMEOUT_MS
+
+// Likely convenient to keep for debugging
+console.log('process.env', process.env)
```

Comment:
Instead of default prod values for packaged apps (outside of the repo dev env), we let the vite mode assign the right ones. This way we can package dev and prod apps.